contrib/examples/rules/sample_rule_file_watch.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: sample_rule_file_watch
+pack: "examples"
+description: Sample rule custom trigger type - add a file to be watched by file_watch_sensor in linux pack.
+enabled: false
+
+trigger:
+  parameters:
+    file_path: /var/log/dmesg
+  type: linux.file_watch.file_path
```

Comment:
The emitted trigger is `linux.file_watch.line` (I checked the logs, too) and here it is `linux.file_watch.file_path`. Funniest is that it works.